File: sdk/Listat/Models/Statistic.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Listat.Models
{
    public class Statistic
    {
        public string Id { get; set; } = string.Empty;

        public DateTimeOffset CreationTime { get; set; }

        public DateTimeOffset ModificationTime { get; set; }

        public string Payload { get; set; } = string.Empty;

        public string Uri { get; set; } = string.Empty;

        public string Category { get; set; } = string.Empty;
    }
}

File: src/models/statistic.go
package models

import (
	"time"
)

type Statistic struct {
	Id               string
	CreationTime     time.Time
	ModificationTime time.Time
	Payload          string
	Uri              string
	Category         string
}

type StatisticQuery struct {
	Id               string
	CreationTime     time.Time
	ModificationTime time.Time
	Payload          string
	Uri              string
	Category         string
	Offset           int
	Limit            int
}
